adapters/src/androidTest/java/io/realm/RealmCursorTest.java

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm;
+
+import android.annotation.TargetApi;
+import android.database.CharArrayBuffer;
+import android.database.ContentObserver;
+import android.database.Cursor;
+import android.database.CursorIndexOutOfBoundsException;
+import android.database.DataSetObserver;
+import android.os.Build;
+import android.os.Bundle;
+import android.test.AndroidTestCase;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import io.realm.entity.AllJavaTypes;
+
+public class RealmCursorTest extends AndroidTestCase {
```

Comment:
All of these should be converted to JUnit4 since that is what we are writing all new tests with. This also include removing the `test` prefix. See https://github.com/realm/realm-java/blob/master/CONTRIBUTING.md#unit-tests